manual/manual/refman/typedecl.etex

```diff
@@ -36,12 +36,14 @@ type-params:
         | '(' type-param { "," type-param } ')'
 ;
 type-param:
-          [variance] "'" ident
+          [variance] [injectivity] "'" ident
```

Comment:
(see the comment attached in `parser.mly` about the order of annotations)

Reply:
Indeed, allowing both orders makes this a pain too.
This said, for things like `private virtual` or `mutable virtual` both orders are allowed, so we maybe should accept the extra complexity.